counter.go

```diff
@@ -95,7 +106,11 @@ func (cv *CounterVector) Get(variableLabels ...string) (*Counter, error) {
 	if cv == nil {
 		return nil, nil
 	}
-	return nil, nil
+	m, err := cv.getOrCreate(variableLabels...)
```

Comment:
this will allocate again right?

Reply:
No, once you have a slice splatting it out is free.